doc/writing.md

```diff
@@ -150,14 +148,14 @@ Hello. ++I have added this part++
 
 ### Line break
 
-You can force line breaks in text (carriage return) using `<br>`, which is the
+You can force line breaks in text (carriage return) using `<br>`, similar to
 [HTML element _line
 break_](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br).
 
 ### Mathematical expressions
 
 Thanks to [KaTeX](https://khan.github.io/KaTeX/), Monod supports mathematical
-expressions. Write LaTeX symbols into `$`:
+expressions. Write LaTeX symbols enclosed within `$`:
```

Comment:
I thought it would be better that way. "Surrounded by" seemed quite informal to me so changed it to "enclosed within". Would change it to as suggested by you.